docs/api/navigators/DrawerNavigator.md

```diff
@@ -91,6 +91,16 @@ The route configs object is a mapping from route name to a route config, which t
 - `contentComponent` - Component to use to render the navigation items. Receives the `navigation` prop for the drawer. Defaults to `DrawerView.Items`.
 - `contentOptions` - Configure the drawer content, see below.
 
+#### Example:
+
+```js
+{
+  drawerWidth: 200,
+  drawerPosition: 'right',
+  contentComponent: props => <ScrollView><DrawerView.Items {...props} /></ScrollView>
+}
```

Comment:
What this example is demonstrating? Please describe. E.g. `The following example demonstrates how you can set a custom `contentComponent`. In this case, we wrap default `DrawerView` with a `ScrollView` so that we can scroll the items in the sidebar.